src/pubsub/index.js

```diff
@@ -328,6 +341,7 @@ class PubsubBaseProtocol extends EventEmitter {
         stream,
         async (source) => {
           for await (const data of source) {
+            // @ts-ignore data slice from BufferList
```

Comment:
add bl types to MuxedStream.source 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bl/index.d.ts